content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm/troubleshooting.md

```diff
@@ -166,23 +165,23 @@ DD_INSTALLER_DEFAULT_PKG_VERSION_DATADOG_APM_INJECT=<VERSION>-1
 
 ### Default deny list
 
-Datadog maintains an internal deny list to prevent injection into certain processes (for example, IDEs or databases). If a process command or entrypoint is on this list, the injector exits the injection process.  
+Datadog maintains an internal deny list to prevent injection into certain processes (for example, IDEs or databases). If a process command or entrypoint is on this list, the injector exits the injection process.
```

Comment:
```suggestion
- Datadog maintains an internal deny list to prevent injection into certain processes (for example, IDEs or databases). If a process command or entrypoint is on this list, the injector exits the injection process.
+Datadog maintains an internal deny list to prevent injection into certain processes (for example, IDEs, or databases). If a process command or entrypoint is on this list, the injector exits the injection process.
```

Fixes Vale Oxford comma issue